OpenUpSA/wazimap-ng-ui
# order of categories, subcategories and indicators in rich data view

## Description
ordering the categories, subcategories and indicators in the rich data view according to the `order` parameter

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-510

## How is it tested automatically?
added test steps into `__tests__/gui/rich_data.feature`

## How should a reviewer test it locally
* in the admin panel change the orders of categories, subcategories and indicators
* confirm that in the rich data panel they are displayed in the correct order
* especially test this with profiles that have multiple versions
* use https://wazimap-pr-433.herokuapp.com as the API url

## Screenshots


## Changelog

### Added

### Updated
* `src/js/profile/profile_loader.js` to order the categories
* `src/js/profile/category.js` to order the subcategories
* `src/js/profile/subcategory.js` to order the indicators

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
